### ISSUE TITLE
Print out STDERR when job submission fails

### ISSUE THREAD

**Issue by jbusecke** [pangeo-forge/deploy-recipe-action#3]
I am getting a lot of errors like this
```
Submitting job...
[54](https://github.com/leap-stc/data-management/actions/runs/4942452173/jobs/8836187762#step:5:55)
recipe_ids = ['METAFLUX_GPP_RECO_monthly']
[55](https://github.com/leap-stc/data-management/actions/runs/4942452173/jobs/8836187762#step:5:56)
Submission jobname = 'METAFLUX_GPP_RECO_monthlybc9fe248'
[56](https://github.com/leap-stc/data-management/actions/runs/4942452173/jobs/8836187762#step:5:57)
Running PGF runner with extra_cmd = ['--Bake.recipe_id=METAFLUX_GPP_RECO_monthly', '--Bake.job_name=METAFLUX_GPP_RECO_monthlybc9fe248']
[57](https://github.com/leap-stc/data-management/actions/runs/4942452173/jobs/8836187762#step:5:58)
  File "/deploy_recipe.py", line 100, in <module>
[58](https://github.com/leap-stc/data-management/actions/runs/4942452173/jobs/8836187762#step:5:59)
    deploy_recipe_cmd(cmd + extra_cmd)
[59](https://github.com/leap-stc/data-management/actions/runs/4942452173/jobs/8836187762#step:5:60)
  File "/deploy_recipe.py", line 17, in deploy_recipe_cmd
[60](https://github.com/leap-stc/data-management/actions/runs/4942452173/jobs/8836187762#step:5:61)
    raise ValueError("Job submission failed.")
```
but there is no information provided about **why** it failed. Could we capture the stderr just like [here](https://github.com/pangeo-forge/deploy-recipe-action/blob/f81c6961b585b7d0002ed73610c8a91cd62dcd6c/action/deploy_recipe.py#L60) and print it in case of failure?

**Comment by jbusecke** [comment]
This seems crucial to debug https://github.com/leap-stc/data-management/actions/runs/5424470779/jobs/9868459154?pr=21#step:5:58

**Comment by cisaacstern** [comment]
Fixed by @jbusecke in #10 🎉
